src/components/Sum.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Sum({ a = 0, b = 0 }) {
+  const sum = a + b;
+
+  return (
+    <p>
+      {`Sum of
+      ${a}
+      and
+      ${b}
+      is
+      ${sum}
+    `}
+    </p>
```

Comment:
I think it looks awful. It's a short one, so maybe write it in one string?)

Reply:
solve it, I just saw it in a hint of the readme.